### DISCUSSION TITLE
Plight of verifying my academic status

### DISCUSSION THREAD

**Discussion by eewkmo** [community/community#43872] [GitHub Education]
Hello, I am a teacher from Jinan University, China. When I join the Global Campus, I have uploaded the proof of my current academic status(in Chinese), which is certificated by Jinan University. But it doesn't work. What can I do to finish the verification?

**Comment by samus-aran** [comment]
Hi @eewkmo  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
